recipes/wagtail-flexible-forms/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "wagtail-flexible-forms" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/wagtail_flexible_forms-{{ version }}.tar.gz
+  sha256: e0b44f5cebca39fbf668969e9523e8899b4d6aa1ffda81347b201c0ab0d715dd
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python ={{ python_min }}
+    - setuptools >=65.0
+    - pip
+  run:
+    - python >=3.8
```

Comment:
If you really need this for py38 you have to add `{% set python_min = "3.8" %}` at the top of your recipe. Just note that conda-forge no longer supports Python <=3.9.

```suggestion
```